Meta/lint-ports.py

```diff
@@ -11,7 +11,7 @@
 # and captures "bash" in group 1, "bash/" in group 2, "<spaces>" in group 3, "GNU Bash" in group 4, "5.0" in group 5
 # and "https://www.gnu.org/software/bash/" in group 6.
 PORT_TABLE_REGEX = re.compile(
-    r'^\| \[`([^`]+)`\]\(([^\)]+)\)([^\|]+) \| ([^\|]+) \| ([^\|]+?) \| ([^\|]+) \|+$', re.MULTILINE
+    r'^\| \[`([^`]+)`\]\(([^\)]+)\)([^\|]*) \| ([^\|]+) \| ([^\|]+?) \| ([^\|]+) \|+$', re.MULTILINE
```

Comment:
Is the regex change here and the array vs non-array change below part of adding the new script? or just general bugfixes? or leftovers from a different approach?

Reply:
itʼs needed if you exactly match the length of the content (thatʼs `"<spaces>" in group 3` from the comment above), i.e. `| ××××× |`, so with a single space from the pipe in the “name” column.

treat it as a general fix; was very important for “snug-in” approach but now it can just happen—no need to alert if it everything fits (that could happen if you provide a description with length exceeding of current minimal width without changing the minimum, i.e. just by working with the port).